firefox-ios/Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -104,6 +104,9 @@ class TelemetryWrapper: TelemetryWrapperProtocol, FeatureFlaggable {
            let uuid = UUID(uuidString: uuidString) {
             GleanMetrics.LegacyIds.clientId.set(uuid)
         }
+        
+        GleanMetrics.Pings.shared.usageDeletionRequest.setEnabled(enabled: true)
+        GleanMetrics.Pings.shared.onboardingOptOut.setEnabled(enabled: true)
```

Comment:
@badboy Do you think this is good spot, before init profile id? I assume we have to do the same for onboarding opt out since it has the same property.

I also wonder, do we ever need to disable them?

Reply:
No disabling needed. It's only explicitly submitted anyway and we can't hit that codepath unless it's the actual disabling by a user through the UI.